lib/brook/dispatcher.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Brook.Dispatcher do
```

Comment:
I actually really like grouping the default implementation module in the same file as the module that defines the behaviour; i'd suggest moving the other defaults (default driver for example) into the same file as their respective behaviour definitions.

Reply:
I don't generally like more than one module per file, but in this case I agree with Jeff; the default adds clarity/an example to the interface.